### DISCUSSION TITLE
Permission to add label to PR

### DISCUSSION THREAD

**Discussion by kkmuffme** [vimeo/psalm#10967] [Psalm development (AKA internals)]
Would it be possible to grant me permission to add labels on (my) PRs e.g. https://github.com/vimeo/psalm/pull/10959 to get the CI to finish without errors.
Since I've more than 100 merged PRs I'm familiar enough with which labels to use

**Comment by weirdan** [comment]
I believe this requires the `Triage` role (https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization).

Let me discuss this with other maintainers.

**Reply by kkmuffme** [reply]
Thanks. 
Is there a Slack/Discord/... the maintainers use or was this the appropriate place to ask this question?

**Reply by weirdan** [reply]
Here is fine. While we do have our private chat on symfony slack, it's just an out-of-band channel for maintainers to communicate, and is in no way an official way to reach us. `symfony-devs#static-analysis` public channel (it's not Psalm-specific) could be of interest for people interested in static analysis in general though.
